.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: GraphScope CI on Local
```

Comment:
Give it a different name, like `Scheduled GraphScope Docker Build CI`

Reply:
This name is already used by `local.yml`

Reply:
fixed